src/app/clusters/level-control/level-control.cpp

```diff
@@ -689,12 +688,12 @@ static void moveHandler(EndpointId endpoint, CommandId commandId, uint8_t moveMo
     case EMBER_ZCL_MOVE_MODE_UP:
         state->increasing  = true;
         state->moveToLevel = state->maxLevel;
-        difference         = static_cast<uint8_t>(state->maxLevel - currentLevel);
+        difference         = static_cast<uint8_t>(state->maxLevel - currentLevel.Value());
```

Comment:
Will crash if currentLevel is null.  Same for a bunch of the other places in this function.

Reply:
done